common_tools/get_dumps.py

```diff
@@ -0,0 +1,40 @@
+import os
+from pathlib import Path
+import argparse
+
+
+def compile_dumps(mlir_dir: str):
+    dumps_dir = Path(args.dir).parent.joinpath("dumps")
```

Comment:
Use the `/` operator for joining dirs:
```suggestion
    dumps_dir = Path(args.dir).parent / "dumps"
```

Reply:
Also below